intdataplane/int_dataplane.go

```diff
@@ -750,3 +752,39 @@ type iptablesTable interface {
 	RemoveChains([]*iptables.Chain)
 	RemoveChainByName(name string)
 }
+
+// msgStringer wraps an API message to customise how we stringify it.  For example, it truncates
+// the lists of members in the (potentially very large) IPSetsUpdate messages.
+type msgStringer struct {
+	msg interface{}
+}
```

Comment:
Is this way of doing things better than just defining and calling a toString() function?   (Genuine question - it quite possibly is, but I haven't got my head around the pros/cons yet!)